nidirect_news/src/Plugin/Block/FeaturedNewsBlock.php

```diff
@@ -18,7 +20,9 @@ class FeaturedNewsBlock extends BlockBase {
    * {@inheritdoc}
    */
   public function build() {
-    $content['featured_news'] = \Drupal::service('nidirect_news.news')->getNewsEmbed('featured_news');
+    $view = \Drupal::service('nidirect_news.news')->getLatestNewsView();
```

Comment:
Would it be better to use DI here ?

Reply:
Yep, in hindsight it couldn't hurt. Means a bit of extra boilerplate to go in but shouldn't take too long.

Reply:
Pushed an update this morning to sort this.